MoEngagePluginCards/DataTransfer/HybridToNative/MoEngageCardsCategoryData.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MoEngageCardsCategoryData.swift
+//  MoEngagePluginCards
+//
+//  Created by Soumya Mahunt on 21/06/23.
+//
+
+import MoEngageCards
+
+struct MoEngageCardsCategoryData {
+    enum HybridKeys {
+        static let category = "category"
+    }
+
+    let category: String
```

Comment:
I kept this model separate considering future design decisions, since the contract might change in future keeping the logic separate allows us to extend functionalities in better ways and follow SOLID principles. What are your thoughts?